lib/inline-imports.js

```diff
@@ -22,17 +23,16 @@ module.exports = function inlineImports(fileContents, options, visited = new Set
   // appropriately large code-base, and potentially leave benchmarks in place
   let basedir = typeof options === 'object' && options !== null && options.basedir;
   if (typeof basedir !== 'string') {
-    throw new Error('inlineImports requires options.basedir be set');
+    throw new Error('inlineImports.lineToImports requires options.basedir be set');
   }
 
-  const result = [];
-  const files = Object.create(null);
   const resolve = typeof options.resolve === 'function' ? options.resolve : require('resolve').sync;
   const fs = typeof options.fs === 'function' ? options.fs : require('fs');
 
-  for (const line of fileContents.split(EOL_REGEXP)) {
-    const matched = matchImport(line);
-
+  const lines = fileContents.split(EOL_REGEXP);
```

Comment:
whats the TL;DR for this change?

Reply:
In the case where we want to know what fragments came from what import (to validate unused imports) we need the line number.

Having access to the line number (via index of the split) is the only motivation for changing the iteration mechanism

Reply:
Just to confirm, you don't need to change I just want to confirm my understand, the previous implementation but with a counter would be the same right?